tests/unit/Trace/TracerTest.php

```diff
@@ -31,8 +31,9 @@ class TracerTest extends TestCase
 {
     private $reporter;
 
-    public function setUp()
+    protected function setUp(): void
     {
+        parent::setUp();
```

Comment:
This shouldn't be nesssary, nothing happens in `setUp` of the parent test case.

Reply:
It's good practice. Avoids nasty surprises if stuff later _does_ get added to the parent implementation of `setUp`.

Reply:
I disagree with this strongly, it's confusing because it implies that something's being called in the `setUp` method. If ever this class gets extended (which is highly unlikely), it's very obvious to the person making the change that such logic would need to be added here (if the extended class contained the same methods). That's basic programming IMHO and does not need an attempt at future-proofing.

Also, this is a test class, so the risk is nonexistant.